sdk/client/src/blocking.rs

```diff
@@ -26,6 +26,8 @@ use serde::{de::DeserializeOwned, Serialize};
 
 const REQUEST_TIMEOUT: u64 = 10_000;
 
+const USER_AGENT: str = "diem-client-sdk-rust / " + env!("CARGO_PKG_VERSION");
```

Comment:
Since we need this in both the the blocking and async clients, but the value is the same, let's stick this in the lib.rs file that way we don't need to duplicate this.

Reply:
Rust has this notion of Sized and !Sized (unsized) types and in most places the compiler needs to work with Sized types (basically all non-heap allocated places). So you'll probably need to change the type of this const to be a `&str` instead of a `str`

Reply:
Since this is a `const` the evaluation of the right hand side needs to be done at compile time using only const functions. Unfortunately `+` isn't a const operations but you should look at using the [concat!](https://doc.rust-lang.org/stable/std/macro.concat.html) macro in place of `+`.

Reply:
Also as a heads up, you may have to wrap the definition of this const with the `cfg_async_or_blocking! { }` macro when you move this to lib.rs. You should be able to see how that macro is already used in lib.rs for an idea of what to do.